### PR TITLE
Multi-versionize py3-rouge-score

### PR DESCRIPTION
**Pull request by justinvreeland**
Fixes:

Related:

### CODE REVIEW

**Comment by justinvreeland** [comment]
@pnasrat  Still can't assign unfortunately.

File: py3-rouge-score.yaml [hunk 1]
```diff
@@ -29,10 +41,22 @@ pipeline:
...
+    name: py${{range.key}}-${{vars.pypi-package}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
 
-  - uses: strip
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
```
Comment (pnasrat) [inline py3-rouge-score.yaml:54]: remove py3-supported block in general do not add this subpackage.

File: py3-rouge-score.yaml [hunk 2]
```diff
@@ -29,10 +41,22 @@ pipeline:
...
-  - name: Python Build
-    uses: python/build-wheel
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
```
Comment (pnasrat) [inline py3-rouge-score.yaml:51]: add a test block wit a test import (probably `rouge_score`)

**Review by pnasrat — CHANGES_REQUESTED** [review]

File: py3-rouge-score.yaml (hunk 1 above)
Comment (justinvreeland) [reply py3-rouge-score.yaml:54]: Don't we need to to provide for packages that will depend on this package once they're also building for multiple versions?

**Review by pnasrat — APPROVED** [review]
LGTM

**Pull request merged**